config/initializers/doorkeeper.rb

```diff
@@ -49,15 +54,27 @@
   # Change the way access token is authenticated from the request object.
   # By default it retrieves first from the `HTTP_AUTHORIZATION` header, then
   # falls back to the `:access_token` or `:bearer_token` params from the `params` object.
-  # Check out the wiki for mor information on customization
+  # Check out the wiki for more information on customization
   # access_token_methods :from_bearer_authorization, :from_access_token_param, :from_bearer_param
 
-  # Change the test redirect uri for client apps
+  # Change the native redirect uri for client apps
   # When clients register with the following redirect uri, they won't be redirected to any server and the authorization code will be displayed within the provider
   # The value can be any string. Use nil to disable this feature. When disabled, clients must provide a valid URL
   # (Similar behaviour: https://developers.google.com/accounts/docs/OAuth2InstalledApp#choosingredirecturi)
   #
-  # test_redirect_uri 'urn:ietf:wg:oauth:2.0:oob'
+  # native_redirect_uri 'urn:ietf:wg:oauth:2.0:oob'
+
```

Comment:
If you expand the diff for this file, these blocks are still in this file.
https://github.com/openstax/accounts/pull/166/files#diff-e621685ac1c2689eaedaedf334c6123bL7